api/db-cdr/generate-cdr/generate-cloudsql-cdr.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# This generates new cloudsql database for a cdr with counts
```

Comment:
I would rename this file and update the comment. This is generating CSV files in GCS, not making a new database in Cloud SQL.

Reply:
will do